labtool2.0/src/components/pages/ManageTags.js

```diff
@@ -202,19 +209,11 @@ export const ManageTags = props => {
                       onChange={(e, { value }) => (state.valueText = value)}
                     />
                   </Form.Field>
-                  <Form.Field required inline>
+                  <Form.Field required>
                     <label style={{ width: '100px', textAlign: 'left' }}>Color</label>
-                    <select className="ui dropdown" value={state.valueColor} name="color" style={{ minWidth: '30em' }} required onChange={e => (state.valueColor = e.target.value)}>
-                      <option value="" disabled>
-                        Select a tag color
-                      </option>
-                      <option value="white">White</option>
-                      {validColors.map(color => (
-                        <option key={color} value={color}>
-                          {capitalize(color)}
-                        </option>
-                      ))}
-                    </select>
+                    <div style={{ display: 'block', marginLeft: '110px', marginRight: '100px' }}>
+                      <SwatchesPicker onChangeComplete={color => (state.valueColor = color.hex)} />
```

Comment:
Maybe we should use a picker that allows to choose *any* color instead of showing a list of options?